crates/edr_op/src/hardfork/mod.rs

```diff
@@ -0,0 +1,45 @@
+use std::sync::OnceLock;
```

Comment:
We don't use `mod.rs`, except for one exception in `tests/integration/mod.rs` (due to compiler limitations for that instance). Could you please make this `crates/edr_op/src/hardfork.rs` to follow that naming convention.